go-controller/pkg/node/management-port-dpu.go

```diff
@@ -192,11 +184,13 @@ func getManagmentPortNetDev(netdevName string) (netlink.Link, error) {
 }
 
 func (mp *managementPortNetdev) Create(nodeAnnotator kube.Annotator, waiter *startupWaiter) (*managementPortConfig, error) {
-	klog.Infof("Lookup netdevice link and existing management port")
-	link, err := getManagmentPortNetDev(mp.netdevName)
```

Comment:
We can get rid of the function `getManagmentPortNetDev()` above this function now right?

Reply:
It is used at `createNodeManagementPorts()`

Reply:
Please fix the spelling of the name `getManagmentPortNetDev()`

Reply:
@wizhaoredhat You're right. We've lost it in between the commits. It can be seen here https://github.com/ovn-org/ovn-kubernetes/pull/3530/commits/9e96d3496c5dd16a17f5601fd6141c491c167195. @bn222 we need it or export function wouldn't work for ovn-k8s-mp0 name. How I see the code - move this function to `default_...go` file, because it is used there only and let it return `link.Attrs().Name`, so the name will match what it does. WDYT?

Reply:
@DmytroLinkin I don't think it's used either in `default_node_network_controller.go` at this point. Do you want to use it somewhere specifically?

Reply:
Yes. It should be used before annotation is exported - in related `if` block or inside `getMgmtPortAndRepName()`.
For You, probably, current PR works, because altnames are used. But for setups without it, initialization will fail when mgmt port already have name `ovn-k8s-mp0`.

Reply:
I've changed the function to use `getManagementPortNetDev()`. Is this what you had in mind?

Reply:
Not really :)
I'll comment there.